### PR TITLE
add landing page hero section, add own style sheet for landing page, …

### PR DESCRIPTION
**Pull request by SamGamgy**
…update global 'body' styles

### CODE REVIEW

**Comment by SamGamgy** [comment]
accidentally altered button.css style sheet. I removed my redundant code and commited for review

File: main/landing-page.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    color: white;
+    font-size: 4.5rem;
+    margin-bottom: 1.5rem;
+}
+.hero-sub-head{
+    text-align: center;
+    color: white;
+    font-size: 1.3rem;
+    font-weight: 200;
+    margin-left: 28%;
```
Comment (mikesz88) [inline main/landing-page.css:25]: It is not recommended to use percentages. Changing this to rems is my recommendation.

File: main/landing-page.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    margin-right: 28%;
```
Comment (mikesz88) [inline main/landing-page.css:26]: It is not recommended to use percentages. Changing this to rems is my recommendation.

File: main/landing-page.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    background-image: url(/BaseCamp-LandingPage/assets/home-page/Typing_Home_Page.jpg);
+    background-position:center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.hero-container{
+    margin-right: auto;
+    margin-left: auto;
+    text-align: center;
+    padding:5% 25%;
```
Comment (mikesz88) [inline main/landing-page.css:13]: It is not recommended to use percentages. Changing this to rems is my recommendation. pixels can work here.

**Review by mikesz88 — CHANGES_REQUESTED** [review]

**Review by mikesz88 — APPROVED** [review]
lgtm

**Pull request merged**